mindmeld/active_learning/results_manager.py

```diff
@@ -0,0 +1,186 @@
+import os
+import json
+import datetime
+import logging
+from typing import Dict, List
+
+from mindmeld.markup import dump_query
+from ..path import (
+    AL_PARAMS_PATH,
+    AL_RESULTS_FOLDER,
+    AL_PLOTS_FOLDER,
+    AL_ACCURACIES_PATH,
+    AL_SELECTED_QUERIES_PATH,
+)
+from ..constants import STRATEGY_ABRIDGED
+
+logger = logging.getLogger(__name__)
+
+
+def create_dir_if_absent(base_path: str):
+    """Create a directory if one doesn't already exist at the given path.
+    Args:
+        base_path (str): Path to create directory."""
+    try:
+        os.makedirs(base_path)
```

Comment:
There is a parameter `exist_ok=True` that does this for you.

Reply:
Removed create_dir_if_absent() and am now using `os.makedir(path, exist_ok=True)` as suggested